api/main.py

```diff
@@ -328,3 +328,15 @@ async def post_regression(regression: Regression,
     await pubsub.publish_cloudevent('regression', {'op': operation,
                                                    'id': str(obj.id)})
     return obj
+
+
+@app.get('/regression/{regression_id}', response_model=Regression)
+async def get_regression(regression_id: str):
```

Comment:
If Regressions are derived from Node, then maybe we could use the nodes API entry points to find them using `kind` set to `regression` and for example `parent_id` or other Node fields to find some particular ones.

Reply:
I don't think we can use `Node` endpoints for `Regression` because `db.find_by_id` and `db.find_by_attributes` will take a model name as the parameter that will differ for `Node` and `Regression` models.

Reply:
At the moment we have:
```python
async def get_node(node_id: str):
    """Get node information from the provided node id"""
    return await db.find_by_id(Node, node_id)
```

We could have:
```
async def get_node(node_id: str, kind: str):
    """Get node information from the provided node id"""
    model = model_from_kind(kind)
    return await db.find_by_id(model, node_id)
```